.storybook/blocks/ComponentDetails.jsx

```diff
@@ -119,6 +157,175 @@ const VersionDetails = ({ tag, data = {}, isDeprecated = false, skipDate = false
+
+	switch(linkType) {
+		case "package": 
+			svgAsset = npmSvg()
+			break;
+		case "repository": 
+			svgAsset = githubSvg()
+			break;
+		case "guidelines": 
+			svgAsset = adobeSvg()
+			break;
+		default: 
+			break;
+	}
+
+	return svgAsset;
+};
+
+/**
+ * Converts the linkType to the subtitle text
+ */
+const convertLinkTypeText = (linkType) => {
+	switch(linkType) {
+		case "package": 
+			linkType = "npm"
+			break;
+		case "repository": 
+			linkType = "GitHub"
+			break;
+		case "guidelines": 
+			linkType = "Spectrum website"
+			break;
+		default: 
+			break;
+	}
+	return linkType;
+};
+
+export const ResourceLinkContent = ({data, linkType=["package", "repository", "guidelines"]}) => {
+	const packageJson = data;
+
+	// componentGuidelinesName
+	let packageName = "";
+	let packageLink = "";
+
+	// componentBetaName: for components with guidelines that are still on spectrum-contributions/beta site
+	let packageAltName = "";
+	let packageAltLink = "";
+	
+	if(linkType === "package") {
+		// NPM package name and link 
+		packageName = packageJson?.name ?? undefined;
+		packageLink = (packageName && typeof packageName !== "undefined") ? `https://npmjs.com/${packageName}` : false;
+	}
+	
+	else if (linkType === "repository") {
+		// repo name and link 
+		packageName = packageJson?.name ? packageJson?.name.split('/').pop() : undefined;
+		packageLink = (packageName && typeof packageName !== "undefined") ? `https://github.com/adobe/spectrum-css/tree/main/components/${packageName}` : false;
+	}
+
+	else if (linkType === "guidelines") {
+		// guidelines site name and link 
+		packageName = packageJson?.componentGuidelinesName ?? undefined;
+		packageLink = (packageName && typeof packageName !== "undefined") ? `https://spectrum.adobe.com/page/${packageName}` : false;
+
+		// internal contributions/beta guidelines name and link
+		packageAltName = (packageName) ? undefined : packageJson?.componentBetaName;
+
+		if (!packageLink) {
+			packageAltLink = (packageAltName && typeof packageAltName !== "undefined") ? `https://spectrum-contributions.corp.adobe.com/page/${packageAltName}` : false;
+			packageLink = packageAltLink;
+		}
+	}
+
+	else {
+		throw new Error(`Are you sure you mean "${linkType}"? Please use a valid link type instead: "package", "repository", "guidelines"`);
+	}
+
+	return (
+		<>
+			{packageLink ? <>
+				<ResourceLink href={packageLink} rel="noopener" className="sb-unstyled">
+					{iconSvgs(linkType)}
+					<ResourceTextWrapper>
+						<div className="spectrum-Heading spectrum-Heading--sizeXS">View {linkType}</div>
+						<div className="spectrum-Body spectrum-Body--sizeS">{convertLinkTypeText(linkType)}</div>
+					</ResourceTextWrapper>
+				</ResourceLink>
+				</>
+				: ""
+			}
+		</>
+	)
+};
+
+/**
+ * Displays the list of relevant component links (to NPM, repo, guidelines, etc). 
+ * 
+ * Usage of this doc block within MDX template(s):
+ *  <ResourceListDetails />
+ */
+export const ResourceListDetails = () => {
```

Comment:
I tried to create this component separate from `<ComponentDetails />`, like `<TaggedReleases />`, so we could import it into the MDX or template files as needed. Maybe that's not necessary, though, and it could just be part of the `ComponentDetails` return. I'm no React expert, so I'd appreciate insight or advice on how I should have or could have done this.

My only concern with what I have currently is that I don't use the `npmData` variable.

Reply:
Can we render the Resource List Details into the Component Details rendering? Semantically this is part of the component details and it does always show up after the details list at the top. It also saves us having to remember to add it to every new page.

Reply:
Sure thing- I'll drop that commit that imports it all separately!

Reply:
`ResourceListDetails` should now be rendered with `ComponentDetails`. This commit also has the `console.warn`: https://github.com/adobe/spectrum-css/pull/3129/commits/8fed17befd48f22212d97f921602a881e988266f